doc/developer/logging.rst

```diff
@@ -502,6 +502,45 @@ General utility formats
    representation for a hexdump.  Non-printable characters are replaced with
    a dot.
 
+.. frrfmt:: %pIN (struct iso_address *)
+
+   ([I]so [N]etwork address) - Format ISO Network Address
+
+   ``%pIN``: :frrfmtout:`01.0203.04O5`
+   ISO Network address is printed as separated byte. The number of byte of the
+   address is embeded in the `iso_net` structure.
+
+   ``%pINl``: :frrfmtout:`01.0203.04O5.0607.0809.1011.1213.14` - long format to
+   print the long version of the ISO Network address which include the System
+   ID and the PSEUDO-ID of the IS-IS system
+
+   Note that the `ISO_ADDR_STRLEN` define gives the total size of the string
+   that could be used in conjunction to snprintfrr. Use like::
+
+     char buf[ISO_ADDR_STRLEN];
+     struct iso_net addr = {.len = 4, .addr = {1, 2, 3, 4}};
+     snprintfrr(buf, ISO_ADDR_STRLEN, "%pIN", &addr);
+
+.. frrfmt:: %pIS (uint8_t *)
+
+   ([I]so [S]ystem ID) - Format ISO System ID
+
+   ``%pIS``: :frrfmtout:`0102.0304.0506`
+
+   ``%pISl``: :frrfmtout:`0102.0304.0506.07` - long format to print the
+   PSEUDO-ID in addition to the System ID
+
+   ``%pISll``: :frrfmtout:`0102.0304.0506.07-08` - long long format to print
```

Comment:
I really think this should be %pSY %pPN and %pLS, they are all 3 different objects. An LSP fragment ID isn't a very long system ID, it's a thing unto itself. I'm sure the same code underneath can be shared. but if it's not much extra work I think it makes sense for these to be identified more distinctly.

Reply:
I agree with you

Reply:
Done